model/Hardware/Hardware.md

```diff
@@ -0,0 +1,34 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Hardware
+
+## Summary
+
+The Hardware Profile provides additional metadata, that is useful for hardware.
+
+## Description
+
+The Hardware namespace defines concepts related to hardware, including virtual hardware
+preparation process, its characteristics, and its access methods.
+
+## Metadata
+
+- id: https://spdx.org/rdf/3.1.0/terms/Hardware
+- name: Hardware
+
+## Profile conformance
+
+For an element collection to be conformant with this profile,
+the following has to hold:
+
+
+1. for every `/Hardware/Hardware` The property listed under the physical are not available if virtualFlag = true
```

Comment:
I think it would be better to have different classes, `VirtualHardware` and `PhysicalHardware` (or whatever you want to name them) and have the corresponding properties in the corresponding classes. You may have an abstract `Hardware` superclass for the common properties.

Reply:
We will discuss this issue with the HBOM group  January 10. What is the main reason for your suggestion?